include/nav-menu.php

```diff
@@ -10,7 +10,33 @@
  * @since 1.7.7
  */
 class PLL_Nav_Menu {
-	public $model, $options;
+	/**
+	 * Stores the plugin options.
+	 *
+	 * @var array
+	 */
+	public $options;
+
+	/**
+	 * Instance of PLL_Model.
+	 *
+	 * @var PLL_Model
+	 */
+	public $model;
+
+	/**
+	 * Theme name.
+	 *
+	 * @var string
+	 */
+	protected $theme;
+
+	/**
+	 * Array of menus in a given language used when auto add pages to menus.
```

Comment:
I don't really know how menu are stored. Nested associative arrays?

Maybe you could point to WordPress documentation or code to help visualize those menus.